traffic_ops/traffic_ops_golang/staticdnsentry/staticdnsentry.go

```diff
@@ -103,6 +104,11 @@ func (staticDNSEntry TOStaticDNSEntry) Validate() error {
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.IPv6)
 	case "CNAME_RECORD":
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.DNSName)
+		address := *staticDNSEntry.Address
+		lastChar := address[len(address)-1:]
+		if lastChar != "." && addressErr == nil {
+			addressErr = fmt.Errorf("must be a valid DNS name. Missing a trailing period at the end")
```

Comment:
how about something like

`CNAME address must have a trailing period.` (or is it called a dot?? not sure.)

that way i don't feel like i made 2 mistakes.

Reply:
Code updated.